Release 9.18.0

clairesunstudio opened the pull request:
## 9.18.0 (07/24/2019)

### Added
- (Patternlab) [ContactRow] DP-15056: Added blocks around contact groups to allow overriding on a field-by-field basis. #684

### Removed
- (Patternlab) [Template] DP-13005: Remove unused/broken template from Mayflower #653

### Changed
- (Patternlab) [ImageLink] DP-9262: Add underline to `:hover` and `:focus` states of image links #678

### Security Updates
- (React) [Security] Bump `lodash-es` from 4.17.11 to 4.17.14 #668
- (React) [Security] Bump `extend` from 3.0.1 to 3.0.2 #580

jesconstantine commented:
Nice work!

I can see your description showing when I add the data in the drupal theme!
![image](https://user-images.githubusercontent.com/3279883/35882541-8e7ee246-0b52-11e8-97ce-fca29cdb076e.png)

And I can see that your description does not show, and callout links still render properly in the drupal theme:
![image](https://user-images.githubusercontent.com/3279883/35882591-af0f2c32-0b52-11e8-98b7-8654d8a52e3d.png)

![image](https://user-images.githubusercontent.com/3279883/35882616-c59d6c20-0b52-11e8-8cc4-a87c5eb6e0e3.png)

Commit:
DP-15056: Update contact-row template

Added blocks around contact fields to allow overriding each on a
field-by-field basis. This prevents having to duplicate markup
in Drupal in order to inject the contact information.

avrilpearl committed:
Merge pull request #676 from massgov/master

Master

dependabot-preview[bot] committed:
[Security] Bump lodash-es from 4.17.11 to 4.17.14 in /react (#668)

Bumps [lodash-es](https://github.com/lodash/lodash) from 4.17.11 to 4.17.14. **This update includes security fixes.**
- [Release notes](https://github.com/lodash/lodash/releases)
- [Commits](https://github.com/lodash/lodash/compare/4.17.11...4.17.14)

Signed-off-by: dependabot-preview[bot] <support@dependabot.com>

dependabot-preview[bot] committed:
[Security] Bump extend from 3.0.1 to 3.0.2 in /react (#580)

Bumps [extend](https://github.com/justmoon/node-extend) from 3.0.1 to 3.0.2. **This update includes security fixes.**
- [Release notes](https://github.com/justmoon/node-extend/releases)
- [Changelog](https://github.com/justmoon/node-extend/blob/master/CHANGELOG.md)
- [Commits](https://github.com/justmoon/node-extend/compare/v3.0.1...v3.0.2)

clair0917 committed:
Patternlab/DP-13005  component display updates (#653)

* DP-13005: remove unused/broken page review template

* DP-13005: backstop and changelog files

* DP-13005: IE11 events layouts

* DP-13005: layout through breakpoints

* DP-13005: updates to press listings

* DP-13005: update site logo

* DP-13005: remove non-working update

lannyheidbreder-current committed:
DP-9262: MF [a11y] No indication of link at hover-over (#678)

* DP-9262: Added :hover and :focus underlining to image links

* DP-9262: Added PR number

lannyheidbreder-current committed:
Merge pull request #677 from massgov/feature/DP-15056--contact-row-blocks

DP-15056: Update contact-row template

clairesunstudio committed:
consolidate changelog
